### PR TITLE
Add edit, delete and add functionality

### PR DESCRIPTION
**Pull request by dicodiaz**
* Add functionality for adding task
* Add functionality for editing task description
* Add functionality for deleting task
* Add functionality for clearing all completed
* Add default to task.completed and task.index
* Add functionality for updating indexes when deleting a task or clearing all completed
* All changes to the list are saved in local storage
* Add function to load two cross fading images
* Moved all images functions to images.js
* Add functionality for emptying the addTask bar after adding a task
* Add functionality for not adding empty tasks and error message

### CODE REVIEW

File: src/Task.js [hunk 1]
```diff
@@ -91,6 +119,30 @@ class Task {
...
+    this.tasks = this.tasks.filter((task) => {
+      if (task.completed) task.element.remove();
+      return !task.completed;
+    });
+    this.tasks.forEach((task, i) => {
+      task.index = i + 1;
+    });
+    this.saveLocalTasks();
+  }
+}
```
Comment (wandji20) [inline src/Task.js:148]: - [x] It does not clear the completed task from the app. You can fix it.

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @dicodiaz 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for this project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Add functionality 
:heavy_check_mark:  Delete functionality
:heavy_check_mark:  Edit functionality 

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved
- [ ] It will be nice to create a new file and move the logic for adding, editing, and updating into it.
- [ ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, etc).


**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Task.js (hunk 1 above)
Comment (dicodiaz) [reply src/Task.js:148]: Hello @wandji20, thanks for the review! It does clear the completed tasks for me, you can try the live version [here](http://dicodiaz.me/to-do-list/dist/index.html).

I also recorded a quick Loom to show you, you can watch it [here](https://www.loom.com/share/5e8b03f061c54ea5ba858cb3bdfae8b2).

Let me know if this is what you meant and how to proceed from here. Thanks in advance.

Comment (wandji20) [reply src/Task.js:148]: @dicodiaz, that is a great video:+1: 
I checked your work again and you met that requirement.
Please make sure to mention this particular change in your pull request when submitting it for re-review.

Comment (Emmyn5600) [reply src/Task.js:148]: @dicodiaz Nice video explaining the project implementation, you met all requirements. @wandji20 I will approve a project on your behalf, everything is fine here.

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @dicodiaz 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- The live link is working properly:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/Task.js (hunk 1 above)
Comment (dicodiaz) [reply src/Task.js:148]: Thanks guys!

**Pull request merged**